experimental/addons/istio/resources/istio-vs.cue

```diff
@@ -0,0 +1,152 @@
+"istio-vs": {
+	annotations: {}
+	attributes: {
+		appliesToWorkloads: ["*"]
+		podDisruptive: false
+	}
+	description: "Handle Traffic Via Istio VirtualService."
+	labels: {}
+	type: "trait"
+}
+
+template: {
+	output: {
+		apiVersion: "networking.istio.io/v1alpha3"
+		kind:       "VirtualService"
```

Comment:
trait definition is outputs. eg:
```cue
outputs: virtualService {
....
}
```

Reply:
Good catch , thank you !
Now vela can successfully render resources.But another issue still exist is that configmap schema can't be generated.